include/seqan3/alignment/pairwise/alignment_range.hpp

```diff
@@ -253,6 +159,119 @@ class alignment_range
 //!\brief Deduces from the passed alignment_executor_type
 template <typename alignment_executor_type>
 alignment_range(alignment_executor_type &&) -> alignment_range<std::remove_reference_t<alignment_executor_type>>;
-//!}
+//!\}
+
+/*!\brief The iterator of seqan3::detail::alignment_range.
+ * \implements std::input_iterator
+ */
+template <typename alignment_executor_type>
+class alignment_range<alignment_executor_type>::alignment_range_iterator
+{
+public:
+    //!\brief Type for distances between iterators.
+    using difference_type = typename alignment_range::difference_type;
+    //!\brief Value type of container elements.
+    using value_type = typename alignment_range::value_type;
+    //!\brief Use reference type defined by container.
+    using reference = typename alignment_range::reference;
+    //!\brief Pointer type is pointer of container element type.
+    using pointer = std::add_pointer_t<value_type>;
+    //!\brief Sets iterator category as input iterator.
+    using iterator_category = std::input_iterator_tag;
+
+    /*!\name Constructors, destructor and assignment
+     * \{
+     */
+    constexpr alignment_range_iterator() noexcept = default; //!< Defaulted.
+    constexpr alignment_range_iterator(alignment_range_iterator const &) noexcept = default; //!< Defaulted.
+    constexpr alignment_range_iterator(alignment_range_iterator &&) noexcept = default; //!< Defaulted.
+    constexpr alignment_range_iterator & operator=(alignment_range_iterator const &) noexcept = default; //!< Defaulted.
+    constexpr alignment_range_iterator & operator=(alignment_range_iterator &&) noexcept = default; //!< Defaulted.
+    ~alignment_range_iterator() = default; //!< Defaulted.
+
+    //!\brief Construct from alignment stream.
+    constexpr alignment_range_iterator(alignment_range & range) : range_ptr(&range)
```

Comment:
This line has non-consistent spacing around `&`:

```suggestion
    constexpr alignment_range_iterator(alignment_range & range) : range_ptr(& range)
```

or 

```suggestion
    constexpr alignment_range_iterator(alignment_range &range) : range_ptr(&range)
```